app/javascript/gobierto_data/webapp/components/landingviz/VisualizationsGrid.vue

```diff
@@ -3,7 +3,7 @@
     v-if="publicVisualizations && publicVisualizations.length"
     class="gobierto-data-visualization--grid"
   >
-    <template v-for="{ items, config, columns, name, id, slug, datasetName } in publicVisualizations">
+    <template v-for="{ config, name, id, slug, nameDataset } in publicVisualizations">
```

Comment:
nameDataset instead of datasetName? Why so? Besides, it's misspelled

Reply:
done!